db/service_updates.yml

```diff
@@ -1,3 +1,14 @@
+- date: '2024-07-17'
+  title: Updating how you recommend trainees for qualified teacher status (QTS) or early years teacher status (EYTS)
+  content: |
+    We’ve updated the way you recommend trainees for QTS or EYTS in Register trainee teachers (Register).
+
+    Your trainees’ degree and placement fields in Register are now ‘mandatory’. You must submit accurate data into these fields before you can recommend your trainees for QTS or EYTS.
+
+    This update applies when you recommend a trainee within their trainee record, and when you recommend multiple trainees at the same time, using the bulk recommend tool.
+
+    If you need help meeting these requirements, email us at becomingateacher@digital.education.gov.uk.
```

Comment:
It's missing the link styling...

![image](https://github.com/user-attachments/assets/a68c8ab6-1618-4a6e-adb6-d7375abfadd9)


Maybe
```suggestion
    If you need help meeting these requirements, email us at <a class='govuk-link'
  href='mailto:becomingateacher@digital.education.gov.uk>becomingateacher@digital.education.gov.uk</a>.
```

Reply:
Thanks 👍 Fixed in the next commit.